backend/sozisel/config/config.exs

```diff
@@ -31,7 +31,9 @@ config :sozisel, Sozisel.JitsiTokenGenerator,
   issuer: "sozisel_app",
   secret_key: "9mrcPkvUfoqzuHrfQfP/ExOynZgpL7w"
 
-config :sozisel, SoziselWeb.Recordings, upload_path: "/tmp/"
+config :sozisel, SoziselWeb.Recordings, upload_path: "/tmp/recordings/"
```

Comment:
And did you check that it won't crash? I remember that it was complaining that some files could not be moved to non existent directories. As far as `/tmp` exists` `/tmp/recordings` won't.

Reply:
And why do we need separate folder for images?

Reply:
Okay, so I'll remove this thing and there will be only `/tmp` folder